internal/pkg/template/template.go

```diff
@@ -437,7 +437,7 @@ func (t *Test) ExpectedOutDir() (filesystem.Fs, error) {
 	return copyFs, nil
 }
 
-func (t *Test) Inputs(provider testhelper.EnvProvider, replaceEnvsFn func(string, testhelper.EnvProvider, string) string, envSeparator string) (map[string]interface{}, error) {
+func (t *Test) Inputs(provider *testtemplateinputs.TestInputsEnvProvider, replaceEnvsFn func(string, testtemplateinputs.TestInputsEnvProvider, string) (string, error), envSeparator string) (map[string]interface{}, error) {
```

Comment:
Sem žádný jiný EnvProvider nepřijde, tak asi není potřeba to zobecňovat a nemusím do ostatních doimplementovávat metodu, kterou nevyužijí.

Reply:
Typ by sa mohol volat `testtemplateinputs.EnvProvider`, zbytocne je to duplicitne.

Ok, moze byt, aj ked sa do toho vnasa zbytocne chaos, ale mozeme to zjednotit neskor.

Reply:
upravím 👍